recipes/winregistry/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "winregistry" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/winregistry-{{ version }}.tar.gz
+  sha256: 942fecad3751c1b78b9e6b0a520266903c3023f104668ce1bdbf381ec993ad8b
+
+build:
+  skip: True  # [not win or py<3.7]
```

Looks like this can be noarch. You need to add a `__win` dependency to `run`
```suggestion
  noarch: python
```

---

The build now fails due to this but I guess it's by design?